src/TitleDetails/AddPieceModal/AddPieceModal.js

```diff
@@ -179,6 +183,25 @@ const AddPieceModal = ({
     onSave();
   }, [change, onSave]);
 
+  const onUnreceivePiece = useCallback(() => {
+    const currentPiece = {
+      ...formValues,
+      checked: true,
+    };
+
+    return unreceive([currentPiece])
+      .then(() => {
+        onStatusChange(PIECE_STATUS.expected);
```

Comment:
If I'm not mistaken, calling "unreceive" API updates the piece status and you don't need to do it again.

Reply:
Let me check it.

Reply:
https://github.com/folio-org/ui-receiving/assets/116072773/2c5b96d8-0f79-4409-bf22-d375ee0a68a6